components/benefit_sponsors/app/models/benefit_sponsors/locations/address.rb

```diff
@@ -102,10 +102,17 @@ def to_html
       #   model.to_s
       #
       # @return [ String ] the full address
-      def to_s
-        city.present? ? city_delim = city + "," : city_delim = city
-        line3 = [city_delim, state, zip].reject(&:nil? || empty?).join(' ')
-        sanitize_html([address_1, address_2, line3].reject(&:nil? || empty?).join('<br/>'))
+      def to_s(use_break_separator: true)
+        city_delim = city.present? ? "#{city}," : city
+        line3 = [city_delim, state, zip].reject(&:blank?).join(' ')
+        sanitize_html([address_1, address_2, line3].reject(&:blank?).join(use_break_separator ? '<br/>' : ', '))
+      end
+
+      # Get the full address formatted as a comma-delimited string
+      #
+      # @return [ String ] the full address
+      def to_s_without_breaks
```

Comment:
Generally want to not be putting view logic in the Model, but legacy already was, and IMO refactoring the above such that we can call it directly and not have to filter out the breaks after the fact as we were doing before this PR is a better approach IMO. Open to suggestions here, maybe moving all of this into a helper instead is better?

Reply:
this in my mind should be in a helper rather than models, but like you said it was already here in legacy so OK for now